openedx/features/edly/utils.py

```diff
@@ -0,0 +1,14 @@
+import jwt
+from django.conf import settings
+
+def encode_edly_user_info_cookie(cookie_data):
+    """
+    Returns JWT encoded string of cookie data
+    """
+    return jwt.encode(cookie_data, settings.EDLY_COOKIE_SECRET_KEY, algorithm='HS256')
```

Comment:
In setting set a value `EDLY_JWT_ALGORITHM = HS256` and use it here like
```
algorithm=settings.EDLY_JWT_ALGORITHM
```

Reply:
done